RepoDb.Extensions/RepoDb.PostgreSql.BulkOperations/RepoDb.PostgreSql.BulkOperations/Compiler.cs

```diff
@@ -732,8 +732,12 @@ private static class ParameterizedMethodFuncCache<TEntity, TResult>
             public static Func<TEntity, object[], TResult> GetFunc(string methodName,
                 Type[] types)
             {
-                var key = methodName.GetHashCode() + types?.Sum(e => e.GetHashCode());
```

Comment:
This change has the added benefit of removing the allocation of an iterator during hash code creation